### PR TITLE
Add Feature: fixed the alignment of footer like flexing

### PR DESCRIPTION
**Pull request by TheBaljitSingh**
fixed the footer section: 
used tailwind flex property with minimum height for proper alignment.  
image

### CODE REVIEW

File: src/layouts/MainLayout.jsx
Comment (k1174) [inline src/layouts/MainLayout.jsx:1]: please formate the code.

**Comment by k1174** [comment]
image
above image for reference for adding space around div so it can get better effect.

**Comment by k1174** [comment]
image

adding footer into a div will work.

**Comment by TheBaljitSingh** [comment]
update the issue related to margin

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,29 +1,32 @@
 const Footer = () => {
     return (
 
+        // <div className="relative w-screen">
```
Comment (k1174) [inline src/components/Footer.jsx:4]: yeh wale div comment hta de.

Comment (TheBaljitSingh) [reply src/components/Footer.jsx]: done

**Pull request merged**